File: Python/PrimeNumbers.py
def isPrime(number):
   for i in range(2,number):
      if number%i==0:
         return False      
   return True
      
print("begin")


print(isPrime(11))
print(isPrime(12))   
   
for i in range(2, 100):
   if isPrime(i):
      print (i)
        
print ("end")       
   
     

File: Java/PrimeNumbers.java
public class PrimeNumbers{

   
   private static boolean isPrime(int number){
   
   		for(int i=2; i<number; i++){
   		    if(number%i==0){
   		         return false;
   		    }
   		
   		}
   		
   		return true;			
   }
   
   
   public static void main(String[] args){
   
      for(int i=2; i<100; i++){
         if(isPrime(i)){
             System.out.print(i+", ");
         
         }
      
      }
      
      System.out.println();
   
   
   }


}